stock_packaging_calculator/models/product_qty_by_packaging_mixin.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Sébastien Alix <sebastien.alix@camptocamp.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+from odoo import api, fields, models
+
+
+class ProductQtyByPackagingMixin(models.AbstractModel):
+    """Allow displaying product qty by packaging."""
+
+    _name = "product.qty_by_packaging.mixin"
```

Comment:
Adding tests for the mixin should be great as it is a transversal feature.

Reply:
The tests just added